### ISSUE TITLE
Mutation name

### ISSUE THREAD

**Issue by joannananana** [freezecoder/mutsneedle#1]
Hi,

Do you think that you could add the option of viewing the name of the mutation that is represented in each dot?

Thanks a lot!

**Comment by freezecoder** [comment]
Hi

Do you mean this is shown as the default view? If you mouse hover over the dot it will show the mutation ID of that field you used e.g. P72R. Am I understanding your request correctly?

**Comment by joannananana** [comment]
No, the problem is that when I hover over the dot it shows only the
quantity (value) and category of the mutation, which was expected, as the
data frame of the mutations imported, consisted of those two columns along
with the coords. I don't understand what my mistake is.

Στις Πέμ, 23 Αυγ 2018 στις 6:24 μ.μ., ο/η freezecoder <
notifications@github.com> έγραψε:

**Comment by freezecoder** [comment]
OK I see the issue here. What I typically do is rerun mutsneedle() with a different R data frame where the 'category' field is the amino acid change field or HGSVSP e.g. P72R. The 'coord' and 'gene' fields remans unchanged but the 'value' field is recalculated as the frequency of the amino acid change.  Because I typically embed these in shiny apps I toggle between category and mutation name. I hope this helps if you try it out.

**Comment by joannananana** [comment]
Thank you very much for the instant response! I will try it and let you
know!

Στις Πέμ, 23 Αυγ 2018 στις 8:46 μ.μ., ο/η freezecoder <
notifications@github.com> έγραψε:
